### DISCUSSION TITLE
SAML custom <SubjectConfirmationData> property

### DISCUSSION THREAD

**Discussion by nocrabs** [keycloak/keycloak#9148] [Q&A]
When integrating with AWS's opensearch kibana SAML proxy, the 'Recipient' attribute of <SubjectConfirmationData> needs a specific value; similar to OKTA's separate configuration of the 'Recipient URL' function.

Now it seems that the 'Recipient' property of <SubjectConfirmationData> shares the configuration of 'Master SAML Processing URL'

**Comment by bs-matil** [comment]
is this a question?

**Reply by nocrabs** [reply]
I think so, I didn’t find a configuration that can customize the \<SubjectConfirmationData\> assertion properties in the keycloak console

**Reply by bs-matil** [reply]
So you would like to change that and provide a PR?

**Reply by nocrabs** [reply]
Sorry, I’m afraid I can’t do anything. I am not a professional developer, I just found this problem during use, and want to find out if there is a relevant configuration to realize this function. So at this stage, is there no relevant configuration that can realize this function?
